packages/ui/src/components/NumberInputV2/index.tsx

```diff
@@ -0,0 +1,424 @@
+import styled from '@emotion/styled'
+import { Icon } from '@ultraviolet/icons'
+import type {
+  ChangeEvent,
+  ForwardedRef,
+  InputHTMLAttributes,
+  ReactNode,
+} from 'react'
+import {
+  createRef,
+  forwardRef,
+  useCallback,
+  useId,
+  useImperativeHandle,
+  useMemo,
+} from 'react'
+import { Button } from '../Button'
+import { Stack } from '../Stack'
+import { Text } from '../Text'
+import { Tooltip } from '../Tooltip'
+
+const SIZES = {
+  small: '30px',
+  medium: '38px',
+  large: '46px',
```

Comment:
I didn't check so it's may be fine but if they are aligned with the new TextInput will it look ok ? height won't be different ?

Reply:
@fabienhebert the height is aligned, there is only 2 px difference compared to design specs due to border